packages/connection-model/lib/model.js

```diff
@@ -1046,14 +1048,23 @@ Connection = AmpersandModel.extend({
 const parseConnectionStringAsPromise = promisify(parseConnectionString);
 
 async function createConnectionFromUrl(url) {
+  // We use resolveMongodbSrv because it understands the load balancer
```

Comment:
What was happening with "mongo3" here, did we get an error or just the `loadBalancer=true` was not added?

Reply:
Yeah, we get an error because it rejects the unknown option.